google/cloud/iot/v1/device_manager.proto

```diff
@@ -216,14 +275,20 @@ message CreateDeviceRegistryRequest {
 message GetDeviceRegistryRequest {
   // The name of the device registry. For example,
   // `projects/example-project/locations/us-central1/registries/my-registry`.
-  string name = 1;
+  string name = 1 [
+    // TODO: Currently unclear if this is required to be fully qualified.
+    (google.api.resource_type) = "DeviceRegistry"
```

Comment:
The comment for `resource_type` in [annotations.proto](https://github.com/googleapis/api-common-protos/blob/796d5f1e5484e41c984d2d124ee86cbb8b6f07b7/google/api/annotations.proto#L67) says "The fully qualified message name..." Why wouldn't these `resource_type` values also be fully-qualified?

Reply:
Thanks for pointing to annotations.proto; yes it is indeed specified :)